wireless_modules/middle_module/boot.py

```diff
@@ -0,0 +1,54 @@
+# Author: Kunj
+# Last updated: 1/08/2020
+
+# Import relevant libraries
+import esp
+import network
+import config
+import time
+
+
+def do_connect(essid, password):
```

Comment:
Having infinite loops turned out to be the reason why I wasn't able to update my ESP32 using ampy or shell and I had to end up finding a way to remove the files that had the infinite loop. This error could have happened due to some other reason but from my knowledge it seems to be the infinite loop. So the 5 attempt (which was just arbitrarily chosen) was just a temporary fix. I was planning to look into getting the infinite loop working later, the priority at this point was to get the Mqtt class and test files out there so that others don't need to reinvent the wheel :)

Reply:
That's very strange. Perhaps @Blake-Haydon had to deal with a similar issue?

Reply:
The [micropython for esp32 documentation](https://docs.micropython.org/en/latest/esp32/quickref.html) even suggests using an infinite loop, so I feel like something else must have been causing your esp32 to lock up. Could you take another look at this?

Reply:
I actually reckon it would be good if we only had one `boot.py` shared by all the modules, since really all it has to do is connect to wifi using an example config, enable debugging, and turn on an LED when completed, which all modules will need to do. What are your thoughts on having `boot.py` up a directory and have both `middle_module` and `mqtt_test` share the one script?